claim/README.md

```diff
@@ -6,17 +6,17 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/claim/README.md
 
 # Connect Agent to Cloud
 
-You can securely connect a Netdata Agent, running on a distributed node, to securely connect to Netdata Cloud. A Space's
+You can securely connect a Netdata Agent, running on a distributed node, to Netdata Cloud. A Space's
 administrator creates a **claiming token**, which is used to add an Agent to their Space via the [Agent-Cloud link
 (ACLK)](/aclk/README.md).
 
 Are you just starting out with Netdata Cloud? See our [get started with
 Cloud](https://learn.netdata.cloud/docs/cloud/get-started) guide for a walkthrough of the process and simplified
 instructions.
 
-Connecting Agents, also referrered as **nodes**, is a security feature in Netdata Cloud. Through the connection process, you demonstrate in a few ways
+Connecting Agents, also referred as **nodes**, is a security feature in Netdata Cloud. Through the connection process, you demonstrate in a few ways
```

Comment:
‘Connecting’ the agents is _NOT_ a security feature, the verification we do to establish proof of ownership is.

To anybody with basic knowledge of security this sounds like we have no idea what we’re talking about.

To someone who is paranoid about security and being tracked, this description is a _huge_ red flag.

This should probably instead be rephrased to something like:

‘When connecting an agent (also referred to as a node) to Netdata Cloud, you must complete a verification process that proves you have some level of authorization to manage the node itself. This verification is a security feature that helps prevent unauthorized users from seeing the data on your node.’

Reply:
Accepted your suggestion @Ferroin , will put it on next update to the PR. thanks